src/ProjectService.Data/Interfaces/IProjectImageRepository.cs

```diff
@@ -7,7 +7,7 @@
 namespace LT.DigitalOffice.ProjectService.Data.Interfaces
 {
     [AutoInject]
-    public interface IImageRepository
+    public interface IProjectImageRepository
```

Comment:
Why do you changed this name? In project exist only project's images.

Reply:
the name of the repo must match the name of the table